src/view3d.cpp

```diff
@@ -448,92 +650,205 @@ void View3D::drawCursor(const TransformState& transStateIn, const V3d& cursorPos
     TransformState transState = transStateIn.translate(offset);
     V3d relCursor = cursorPos - offset;
 
-    // Cull if behind camera
-    if((relCursor * transState.modelViewMatrix).z > 0)
-        return;
+    // Cull if behind camera -> doesn't work for now
+    //if((relCursor * transState.projMatrix).z > 0)
+    //    return;
```

Comment:
There was an issue with this check always returning `TRUE` and hence not rendering the cursor, because of some uninitialised variable somewhere. I think I fixed that along the way, but didn't go back try out this culling again. However, I don't see how the cursor could be behind the camera, though? Am I missing some interaction pattern here - the camera always centres around the cursor for me.